registration-processor/pom.xml

```diff
@@ -129,7 +129,7 @@
 		<kernel.qrcodegenerator.zxing.version>1.2.1-SNAPSHOT</kernel.qrcodegenerator.zxing.version>
 		<kernel.logger.logback.version>1.2.1-SNAPSHOT</kernel.logger.logback.version>
 		<kernel.idvalidator.vid.version>1.2.1-SNAPSHOT</kernel.idvalidator.vid.version>
-		<object.store.khazana.version>1.2.0.1-SNAPSHOT</object.store.khazana.version>
+		<object.store.khazana.version>1.2.1-SNAPSHOT</object.store.khazana.version>
```

Comment:
Please take changes from 'git pull upstream develop' to get latest changes from develop

Reply:
done